src/main/java/com/github/osisoft/ocs_sample_library_preview/sds/SdsSummaryType.java

```diff
@@ -0,0 +1,22 @@
+package com.github.osisoft.ocs_sample_library_preview.sds;
+
+public enum SdsSummaryType {
+
+    None("None"), Count("Count"), Minimum("Minimum"),
```

Comment:
I don't feel very strongly about it but I think readability would be a little better with each of these values on its own line. Same goes for SummaryDirection.java.

Reply:
I'll change it. It was another thing I was wondering if anyone would comment on. I wanted them on multiple lines, but went with consistency instead and matched the other enum classes.

Reply:
I didn't realize we did it this way in the other enum classes, I get why you would want to be consistent in that case. Still, I'd probably prefer to move towards putting them on their own lines for better readability.

Reply:
i made the change, and the library built successfully. so we should be good. i couldnt test it against the sample itself as a triple check because of the problem with data views right now.